scripts/galaxykickstart_from_workflow.py

```diff
@@ -0,0 +1,96 @@
+                                   "Christophe Antoniewski <drosofff@gmail.com>\n"
+                                   "https://github.com/ARTbio/ansible-artimed/tree/master/scritps/galaxykickstart_from_workflow.py")
+    parser.add_argument('-w', '--workflow',
+                        dest="workflow_files",
+                        required=True,
+                        nargs='+',
+                        help='A comma-separated of galaxy workflow description files in json format', )
+    parser.add_argument('-l', '--panel_label',
+                        dest='panel_label',
+                        default='Tools from workflows',
+                        help='The name of the panel where the tools will show up in Galaxy.'
+                             'If not specified: "Tools from workflows"')
+    return parser.parse_args()
+
+def makedir (path):
+    try:
+        os.makedirs(path)
+    except OSError:
+        if not os.path.isdir(path):
+            raise
+
+def make_inventory (path="../inventory_files/"):
+    file_path = path + "gks_workflows"
+    try:
+        f = os.fdopen(os.open(file_path, os.O_CREAT | os.O_WRONLY | os.O_EXCL), 'w')
+    except OSError:
+        sys.exit("gks_workflows inventory file already exists in ../inventory_files")
+    print >> f, textwrap.dedent('''\
+            [gks_workflows]
+            localhost ansible_connection=local
+
+            # if remote target, replace the above line by
+            # <remote host IP> ansible_ssh_user="root" ansible_ssh_private_key_file="<path/to/your/private/key>"
+            ''')
+    f.close()
+
+def make_gks_workflows_groupvars (workflow_file_list, path="../group_vars/gks_workflows",
+                                  template="galaxykickstart_from_workflow_templates/group_vars_template.yml"):
+    if os.path.exists(path):
+        sys.exit("../group_vars/gks_workflows already exists.")
+    with open (template, "r") as f:
+        with open (path, "w") as o:
+            for line in f:
+                o.write(line)
+            for workflow in workflow_file_list:
+                workflow = os.path.basename(workflow)
+                workflow = "extra-files/gks_workflows/" + workflow
+                o.write ('  - "')
+                o.write (workflow)
+                o.write ('"\n')
+
+
+def make_gks_workflows_extra_files (workflow_files, panel_label, tool_list_file, extra_files_dir="../extra-files/gks_workflows"):
+    makedir(extra_files_dir)
+    copyfile("galaxykickstart_from_workflow_templates/tool_sheds_conf.xml.sample", extra_files_dir + "/tool_sheds_conf.xml.sample")
+    generate_tool_list_from_workflow(workflow_files, panel_label, tool_list_file)
+    copyfile(tool_list_file, extra_files_dir + "/" + tool_list_file)
+    for file in workflow_files:
+        file = os.path.basename(file)
+        copyfile (file, extra_files_dir + "/" + file)
+
+
+
+
+
+def __main__(workflow_file_list, panel_label, tool_list_file):
```

Comment:
I think you could just give this a regular function (create_flavor/create_variant ?), since I can see how this could be useful an imported function as well.